openedx/core/djangoapps/discussions/views.py

```diff
@@ -0,0 +1,120 @@
+"""
+Handle view-logic for the djangoapp
+"""
+from edx_rest_framework_extensions.auth.jwt.authentication import JwtAuthentication
+from edx_rest_framework_extensions.auth.session.authentication import SessionAuthenticationAllowInactiveUser
+from opaque_keys.edx.keys import CourseKey
+from opaque_keys import InvalidKeyError
+from rest_framework import serializers
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from openedx.core.lib.api.permissions import IsStaff
+
+from .models import DiscussionsConfiguration
+
+
+PROVIDER_FEATURE_MAP = {
+    'cs_comments_service': [
+        'discussion-page',
+        'embedded-course-sections',
+        'lti',
```

Comment:
I think this should probably be `lti-1.1` or similar to distinguish between different LTI versions.

Reply:
@xitij2000 May not hurt, though does the frontend need to concern itself with which version of LTI is implemented?

My thought was that this would be used only for the feature chart and other high-level display and _not_ with the actual wiring up of LTI; that's up to the `lti_configuration`.

I figured this would be used to, at best, update the feature chart, at worst, tell the frontend "pass them along to the LTI view when rendering", but still not be concern with the actual version.

We can also add it later, though open to including now if we think it's pertinent.

Reply:
@stvstnfrd Good point! If the provider uses and unsupported version of LTI then, it simply won't be on this list, and if it's supported then simply saying LTI does seem like it's enough.